### ISSUE TITLE
Extension: I/O of RDF data in its various formats

### ISSUE THREAD

**Issue by robert-haas** [trueagi-io/hyperon-experimental#538]
Would a MeTTa extension be desirable for import and export of [RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework) data in its various formats ([.jsonld](https://en.wikipedia.org/w/index.php?title=JSON-LD), [.nt](https://en.wikipedia.org/w/index.php?title=N-Triples), [.nq](https://en.wikipedia.org/w/index.php?title=N-Triples#N-Quads), [.n3](https://en.wikipedia.org/w/index.php?title=Notation3), [.trig](https://en.wikipedia.org/w/index.php?title=TriG_(syntax)), [.ttl](https://en.wikipedia.org/w/index.php?title=Turtle_(syntax)), ...)? Should it be part of the standard library?

Examples from graph databases:
- [Neo4j](https://neo4j.com) has the [NSMNTX](https://neo4j.com/nsmtx-rdf) extension to import RDF data and construct RDF endpoints
- [GraphDB](https://graphdb.ontotext.com) is built with "RDF and SPARQL support" in mind and accordingly provides multiple ways of [Loading](https://graphdb.ontotext.com/documentation/9.11/enterprise/loading-data.html) and [Exporting](https://graphdb.ontotext.com/documentation/9.11/enterprise/exporting-data.html) RDF data

**Comment by Necr0x0Der** [comment]
Yes, it is desirable. I wouldn't put it into `stdlib`, though. We've discussed the possibility to implement importing libraries / modules / packages / extensions from github, and to have a special repo for such libraries or/and registry, which contains a list of curated libraries / repos. Basically, @luketpeterson already implemented a possibility to put MeTTa-libraries into a subfolder for REPL, and we wanted to extend this for Python extensions and to make non-local modules discoverable from the registry (either by `import!` or by a cli tool like `pip`, let it be `mim` - "MeTTa/Mim installs modules" or whatever else). Luke is now working on a module system, which should include all these features at some point. Absence of the package manager not a blocker to develop an extension. It will only require manual installation for now. In fact, Python extension can be represented as a Python package, and in this case `extend-py!` should work whatever the package is importable by Python itself.

**Comment by vsbogd** [comment]
Depends on #470
